dist/homework2.js

```diff
@@ -0,0 +1,126 @@
+"use strict";
```

Comment:
it would be good to name files same as in the original course repository, so the diff would be visible - 
`lesson2-homework.ts` instead of `homework2.ts`